eng/pipelines/coreclr/templates/perf-job.yml

```
@@ -196,7 +196,7 @@ jobs:
           mkdir -p $(librariesDownloadDir)/bin/wasm/dotnet &&
           cp -r $(librariesDownloadDir)/BrowserWasm/staging/dotnet-latest/* $(librariesDownloadDir)/bin/wasm/dotnet &&
           cp -r $(librariesDownloadDir)/BrowserWasm/staging/built-nugets $(librariesDownloadDir)/bin/wasm &&
-          cp src/mono/wasm/test-main.js $(librariesDownloadDir)/bin/wasm/wasm-data/test-main.js &&
+          cp src/mono/wasm/test-main.mjs $(librariesDownloadDir)/bin/wasm/wasm-data/test-main.mjs &&
```

This needs other changes. For this PR make this:
```suggestion
          cp src/mono/wasm/test-main.mjs $(librariesDownloadDir)/bin/wasm/wasm-data/test-main.js &&
```

---

and same for the other performance ones

---

`perf-job.yml` & `run-scenarios-job.yml`, right? How can I validate it?

---

You will need to run `dotnet-runtime-perf`. But I can tell you that it won't work because a change is needed in performance repo too. And *possibly* in bdn.
Copying the `.mjs` to `.js` should be fine though, because we pass `--module` to v8.